test/functional/test_framework/key.py

```diff
@@ -112,40 +112,16 @@ def set_secretbytes(self, secret):
         ssl.BN_CTX_free(ctx)
         return self.k
 
-    def set_privkey(self, key):
```

Comment:
I can imagine these methods potentially being useful at some point. I don't think there's any need to remove them.